Engine/src/neat/neat-genome.cpp

```diff
@@ -153,5 +153,18 @@ void Genome::MutateRemoveLink() {
     RemoveLink(idToRemove);
   }
 }
+  
+void Genome::MutateChangeWeight() {
+  std::random_device rd;
+  std::mt19937 gen(rd());
+  std::uniform_real_distribution<> dis(0.0, mutation_rate);
```

Comment:
Shouldn't this be `dis(0.0, 1.0)`?

Reply:
And I think a Gaussian distribution should be better?